packages/x-date-pickers/src/internals/components/PickersToolbarText.tsx

```diff
@@ -21,10 +21,9 @@ export interface PickersToolbarTextProps
   value: React.ReactNode;
 }
 
-const useUtilityClasses = (ownerState: PickersToolbarTextProps) => {
-  const { classes, selected } = ownerState;
+const useUtilityClasses = (classes: Partial<PickersToolbarTextClasses> | undefined) => {
   const slots = {
-    root: ['root', selected && 'selected'],
+    root: ['root'],
```

Comment:
I'm dropping this state class in favor of a data attribute.
We will probably drop most of them since Base UI (and therefore `@mui/material`) is migrating toward data attributes. And dropping this one allowed me to not create a custom `ownerState` for this component...